src/utils/rewards.ts

```diff
@@ -1,22 +1,22 @@
-import { GovernanceCurrency, StakedBalance } from "../types";
+import { RewardCurrency, StakedBalance } from "../types";
 import Big from "big.js";
 import { MonetaryAmount } from "@interlay/monetary-js";
 import { ATOMIC_UNIT, newMonetaryAmount } from "./currency";
 
 // TODO: simplify this, perhaps use builder?
 export function estimateReward(
```

Comment:
I just realised that this is only used for staking rewards which will always be in governance currency only, so this change should be reverted. Sorry about this!